CMakeLists.txt

```diff
@@ -120,11 +138,51 @@ configure_file(${CMAKE_SOURCE_DIR}/src/rajaperf_config.hpp.in
 
 include_directories($<BUILD_INTERFACE:${PROJECT_BINARY_DIR}/bin>)
 
-# Make sure RAJA flag propagate (we need to do some house cleaning to
+# Make sure RAJA flags propagate (we need to do some tidying to
 # remove project-specific CMake variables that are no longer needed)
 set (CUDA_NVCC_FLAGS ${RAJA_NVCC_FLAGS})
 
 #
 # Each directory in the perf suite has its own CMakeLists.txt file.
 #
+
+# The statement below is required for Kokkos compilation.
+if(ENABLE_KOKKOS)
+  include_directories(SYSTEM ${CMAKE_CURRENT_SOURCE_DIR}/tpl/RAJA/include/)
+endif()
+
+# ENABLE_KOKKOS is A RAJAPerf Suite Option
+if(ENABLE_KOKKOS)
+  add_definitions(-DRUN_KOKKOS)
+  if(ENABLE_HIP)
+    set(Kokkos_ENABLE_HIP ON CACHE BOOL "Kokkos builds for AMD HIP set the
+Kokkos_ENABLE_HIP variable to ON")
+  endif()
+
+  if(ENABLE_TARGET_OPENMP)
+    set(Kokkos_ENABLE_OPENMPTARGET ON CACHE BOOL "Docstring")
+    set(RAJA_ENABLE_TARGET_OPENMP ON CACHE BOOL "Docstring")
```

Comment:
Doesn't thin get set elsewhere based on the value of ENABLE_TARGET_OPENMP?

Reply:
Hi @davidbeckingsale , I don't see another place where `Kokkos_ENABLE_OPENMPTARGET` is set.  I'm guessing that the "this" in your comment refers to the Kokkos option for OpenMPTarget builds, if I am understanding you....?  The `ENABLE_TARGET_OPENMP` RAJAPerf Suite option controls the Kokkos corollary in the top-level CMakeLists.txt file, and not in the `src/CMakeLists.txt` file.   Were you expecting `Kokkos_OPENMPTARGET` to be set elsewhere, and/or controlled differently?